src/rendergraph/CMakeLists.txt

```diff
@@ -39,5 +39,8 @@ set(
 )
 
 add_subdirectory(opengl)
-add_subdirectory(scenegraph)
+#################################
+# TODO: uncomment in follow-up PR
+# add_subdirectory(scenegraph)
+#################################
```

Comment:
Looks like this is the culprit that breaks it for me. WIthout `scenegraph`, the calls that reference the non-existing `rendergraph_sg` target fail.

Side note: Splitting the CMake code over multiple small `CMakeLists.txt` files that reference targets and variables defined elsewhere makes the control flow quite difficult to follow. Have you considered just using a single `CMakeLists.txt` inside the `rendergraph` directory?

Reply:
Ok, for now the calls that reference the non-existing rendergraph_sg target fail should be commented out. I will do that.

I obsoletely prefer splitting cmake over multiple files, one per target.

Reply:
Could you please try again, @Holzhaus ? I added conditionals to only add the shaders for the targets that were added.
(It looks like this is a change in Qt, being more strict about this in more recent versions)